setup_test.go

```diff
@@ -382,6 +382,51 @@ func TestCaddyParse(t *testing.T) {
 				},
 			},
 		},
+		{
+			`
+			txtdirect {
+				enable host qr
```

Comment:
In my view enable should only be used to enable routing/type features not general features. Aka we don't enable prometheus, but we do enable host and path.

So enable qr shouldn't be needed.